plugins/aea-ledger-fetchai/aea_ledger_fetchai/_cosmos.py

```diff
@@ -1020,72 +960,111 @@ def get_transfer_transaction(  # pylint: disable=arguments-differ
         :param gas: the gas used.
         :param memo: memo to include in tx.
         :param chain_id: the chain ID of the transaction.
+        :param account_number: Account number
+        :param sequence: Sequence
         :param kwargs: keyword arguments.
         :return: the transfer transaction
         """
         denom = denom if denom is not None else self.denom
-        chain_id = chain_id if chain_id is not None else self.chain_id
-        account_number, sequence = self._try_get_account_number_and_sequence(
-            sender_address
+
+        amount_coins = [Coin(denom=denom, amount=str(amount))]
+        tx_fee_coins = [Coin(denom=denom, amount=str(tx_fee))]
+
+        msg_send = MsgSend(
+            from_address=str(sender_address),
+            to_address=str(destination_address),
+            amount=amount_coins,
         )
+        send_msg_packed = ProtoAny()
+        send_msg_packed.Pack(msg_send, type_url_prefix="/")
+
+        chain_id = chain_id if chain_id is not None else self.chain_id
+
         if account_number is None or sequence is None:
-            return None  # pragma: nocover
-        transfer_msg = {
-            "type": "cosmos-sdk/MsgSend",
-            "value": {
-                "amount": [{"amount": str(amount), "denom": denom}],
-                "from_address": sender_address,
-                "to_address": destination_address,
-            },
-        }
+            account_number, sequence = self._try_get_account_number_and_sequence(
+                sender_address
+            )
+            if account_number is None or sequence is None:
+                return None  # pragma: nocover
+
         tx = self._get_transaction(
-            account_number,
-            chain_id,
-            tx_fee,
-            denom,
-            gas,
-            memo,
-            sequence,
-            msg=transfer_msg,
+            account_numbers=[account_number],
+            from_addresses=[str(sender_address)],
+            pub_keys=[b""],
+            chain_id=chain_id,
+            tx_fee=tx_fee_coins,
+            gas=gas,
+            memo=memo,
+            sequences=[sequence],
+            msgs=[send_msg_packed],
         )
         return tx
 
     @staticmethod
     def _get_transaction(
-        account_number: int,
+        account_numbers: List[int],
+        from_addresses: List[str],
+        pub_keys: List[bytes],
         chain_id: str,
-        tx_fee: int,
-        denom: str,
+        tx_fee: List[Coin],
         gas: int,
         memo: str,
-        sequence: int,
-        msg: Dict[str, Collection[str]],
+        sequences: [int],
+        msgs: List[ProtoAny],
```

Comment:
Clarify the diffs:

account numberS
from_addresses
public_keys
denom(missing)
sequenceS
msgS